src/index.html

```diff
@@ -10,7 +10,31 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="page">
+    <form
+      action="#"
+      method="get"
+      class="search-bar--big section"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input input--big"
+      >
+    </form>
+    <form
+      action="#"
+      method="get"
+      class="search-bar--small section"
+      data-qa="small">
```

Comment:
```suggestion
    <form
      action="#"
      method="get"
      class="search-bar--small section"
      data-qa="small"
    >
```
5'th point in checklist